src/client/theme-default/components/NavLink.vue

```diff
@@ -7,16 +7,16 @@
 </template>
 
 <script setup lang="ts">
-import { defineProps } from 'vue'
+import { defineProps, toRefs } from 'vue'
 import type { DefaultTheme } from '../config'
 import { useNavLink } from '../composables/navLink'
 import OutboundLink from './icons/OutboundLink.vue'
 
-const { item } = defineProps<{
-  item: DefaultTheme.NavItemWithLink
-}>()
-
-const { props: linkProps, isExternal } = useNavLink(item)
+const props = defineProps<{
+  item: DefaultTheme.NavItemWithLink,
+}>();
+const propsRefs = toRefs(props);
+const { props: linkProps, isExternal } = useNavLink(propsRefs.item);
```

Comment:
I think we only need to stop destructuring. So following should work. Could you test with one? It's working on my env.

```suggestion
const props = defineProps<{
  item: DefaultTheme.NavItemWithLink,
}>();

const { props: linkProps, isExternal } = useNavLink(props.item);
```